### PR TITLE
Club weapon balance (Part 3 of melee weapon balance project)

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary

```SUMMARY: Balance "Rebalances club weapons"```

#### Purpose of change
Mlangsdorf has been doing some deep analysis of the melee weapons and both analysis of dps of weapons and evaluating their strength ingame, and started a large "melee weapon balance" project. His analysis and his proposed values can be found here:
https://docs.google.com/spreadsheets/d/14eQIe4AO_f6OxCt1XcB4NLAs6-5R1wQW-ydZG0orYdY/edit#gid=1655515751
He wanted someone to start implementing the new values ingame, so i started work.

#### Describe the solution
Rebalances club weapons according to excel sheet.
#### Additional context
Credit to Mlangsdorf for the excellent analysis and for most of the proposed values.
Credit to vastin for review of the excel sheet.
Credit to hymore24/hirmuolio for the initial complaint.

### CODE REVIEW

File: data/json/items/melee/bludgeons.json [hunk 1]
```diff
@@ -1053,9 +1050,9 @@
     "techniques": [ "WBLOCK_1", "BRUTAL", "SWEEP" ],
     "flags": [ "DURABLE_MELEE", "SPEAR", "NONCONDUCTIVE", "BELT_CLIP" ],
     "volume": "1250 ml",
-    "bashing": 20,
-    "cutting": 20,
+    "bashing": 22,
+    "cutting": 23,
     "price": 16000,
-    "qualities": [ [ "HAMMER", 1 ] ]
+    "qualities": [ [  "HAMMER", 1 ] ]
```
Comment (LaVeyanFiend) [inline data/json/items/melee/bludgeons.json:1056]:
```suggestion
    "qualities": [ [ "HAMMER", 1 ] ]
```
JSON validator doesn't like doublespace

Comment (Brian-Otten) [reply data/json/items/melee/bludgeons.json]: Thank you.

**Comment by SirPendrak** [comment]
Maybe warhammer should get stabbing instead of cutting?

**Comment by Brian-Otten** [comment]
I thought that as well but when i spawned it ingame it seemed to have stabbing already (i have no clue why). so problem mysteriously solved?

**Pull request merged**